WalletWasabi.Fluent/Views/Dialogs/Authorization/PasswordAuthDialogView.axaml

```diff
@@ -15,7 +15,7 @@
                  CancelContent="Cancel"
                  EnableCancel="{Binding EnableCancel}"
                  EnableBack="{Binding EnableBack}"
-                 EnableNext="True" NextContent="Continue"
+                 EnableNext="True" NextContent="SEND"
```

Comment:
This is not good, this view is used also before Wallet Info where `SEND` doesn't make sense